### PR TITLE
[14.0][MIG] hr_operating_unit

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

File: hr_operating_unit/models/hr_operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import fields, models
+
+
+class HrOperatingUnit(models.Model):
+    _inherit = "operating.unit"
+
+    employee_ids = fields.Many2many(
+        "hr.employee",
+        "operating_unit_employees_rel",
+        "poid",
```
Comment (patrickrwilson) [inline hr_operating_unit/models/hr_operating_unit.py]: This line was mentioned in the v12 PR that it shouldn't be needed in future versions.

File: hr_operating_unit/models/hr_employee.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    operating_unit_ids = fields.Many2many(
+        "operating.unit",
+        "operating_unit_employees_rel",
+        "employee_id",
+        "poid",
```
Comment (patrickrwilson) [inline hr_operating_unit/models/hr_employee.py]: This line was mentioned in the v12 PR that it shouldn't be needed in future versions.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Please see comments on the v12 PR related to the 'poid' that can be removed in future versions.

File: hr_operating_unit/models/hr_operating_unit.py (hunk 1 above)
Comment (Murtaza-OSI) [reply hr_operating_unit/models/hr_operating_unit.py]: Resolve

File: hr_operating_unit/models/hr_operating_unit.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrOperatingUnit(models.Model):
```
Comment (kittiu) [inline hr_operating_unit/models/hr_operating_unit.py]: Non blocking. But I think it is more proper.
```suggestion
class OperatingUnit(models.Model):
```

**Review by kittiu — APPROVED** [review]

**Review by patrickrwilson — APPROVED** [review]
Works Great, LGTM

**Pull request merged**